src/ask.js

```diff
@@ -2,7 +2,8 @@ exports.handler = (event, context, callback) => {
   let axios = require('axios')
   let qs = require('querystring')
 
-  let { text = 'babel eslint error' } = qs.parse(event.body)
+  //trim off white space from the body
+  let { text } = qs.parse(event.body.trim())
```

Comment:
oh you can't call `trim` on `event.body`.. the body contains many keys in the form `foo=1&bar=2`